sql/updates/world/master/9999_99_99_01_world.sql

```diff
@@ -0,0 +1,76 @@
+(@CGUID+11, 148242, 1, 1637, 11386, '0', '0', 0, 0, 0, 1459.3541259765625, -4481.42529296875, 20.06750679016113281, 0.490184634923934936, 120, 0, 0, 4945, 4393, 0, 0, 0, 0, 36839); -- Silvermoon Magister
+
+DELETE FROM `creature_addon` WHERE `guid` BETWEEN @CGUID+1 AND @CGUID+11;
+INSERT INTO `creature_addon` (`guid`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `aiAnimKit`, `movementAnimKit`, `meleeAnimKit`, `auras`) VALUES
+(@CGUID+1, 0, 0, 0, 1, 0, 0, 0, 0, ''), -- Use the Portal to Zuldazar
+(@CGUID+2, 0, 0, 0, 257, 0, 0, 0, 0, '287470'), -- Zandalari Arcanital - 287470
+(@CGUID+3, 0, 0, 0, 257, 0, 0, 0, 0, '287470'), -- Zandalari Arcanital - 287470
+(@CGUID+4, 0, 0, 0, 257, 0, 0, 0, 0, '288451'), -- Shal'dorei Arcanist - 288451
+(@CGUID+5, 0, 0, 0, 257, 0, 0, 0, 0, '288451'), -- Shal'dorei Arcanist - 288451
+(@CGUID+6, 0, 0, 0, 257, 0, 0, 0, 0, '286709'), -- Huojin Magister - 286709
+(@CGUID+7, 0, 0, 0, 257, 0, 0, 0, 0, '286709'), -- Huojin Magister - 286709
+(@CGUID+8, 0, 0, 0, 257, 0, 0, 0, 0, '289141'), -- Argent Mage - 289141
+(@CGUID+9, 0, 0, 0, 257, 0, 0, 0, 0, '289141'), -- Argent Mage - 289141
+(@CGUID+10, 0, 0, 0, 257, 0, 0, 0, 0, '287432'), -- Silvermoon Magistrix
+(@CGUID+11, 0, 0, 0, 257, 0, 0, 0, 0, '287432'); -- Silvermoon Magister
+
+DELETE FROM `creature_template_addon` WHERE `entry` IN (145714,148265,148768,148263,149258,148250,148242);
+INSERT INTO `creature_template_addon` (`entry`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `aiAnimKit`, `movementAnimKit`, `meleeAnimKit`, `auras`) VALUES
+(145714, 0, 0, 0, 1, 0, 0, 0, 0, ''), -- 145714 (Use the Portal to Zuldazar)
+(148265, 0, 0, 0, 257, 0, 0, 0, 0, '287470'), -- 148265 (Zandalari Arcanital)
+(148768, 0, 0, 0, 257, 0, 0, 0, 0, '288451'), -- 148768 (Shal'dorei Arcanist)
+(148263, 0, 0, 0, 257, 0, 0, 0, 0, '286709'), -- 148263 (Huojin Magister)
+(149258, 0, 0, 0, 257, 0, 0, 0, 0, '289141'), -- 149258 (Argent Mage)
+(148250, 0, 0, 0, 257, 0, 0, 0, 0, '287432'), -- 148250 (Silvermoon Magistrix)
+(148242, 0, 0, 0, 257, 0, 0, 0, 0, '287432'); -- 148242 (Silvermoon Magister)
+
+UPDATE `creature_template` SET `minlevel`=45, `maxlevel`=45 WHERE `entry`=148768; -- Shal'dorei Arcanist
+UPDATE `creature_template` SET `minlevel`=50, `maxlevel`=50 WHERE `entry`=145714; -- Use the Portal to Zuldazar
+UPDATE `creature_template` SET `minlevel`=50, `maxlevel`=50 WHERE `entry`=148265; -- Zandalari Arcanital
+UPDATE `creature_template` SET `minlevel`=35, `maxlevel`=35 WHERE `entry`=148263; -- Huojin Magister
+UPDATE `creature_template` SET `minlevel`=30, `maxlevel`=30 WHERE `entry`=149258; -- Argent Mage
+UPDATE `creature_template` SET `minlevel`=50, `maxlevel`=50 WHERE `entry`=148250; -- Silvermoon Magistrix
+UPDATE `creature_template` SET `minlevel`=50, `maxlevel`=50 WHERE `entry`=148242; -- Silvermoon Magister
```

Comment:
Please add NewLine at the end of the file (or start all SQL files consistently with a blank comment line)